kpi/003.adoc

```diff
@@ -42,7 +42,7 @@ in support of data access, visualization and exploitation.
 |-1
 |===
 
-Total possible score: 3 (100%)
+*Total possible score: 3 (100%)*
```

Comment:
line 41 and 42: "abstract contains bulletin template | -1 "

I recommend making this a positive test, for example: "abstract does not contain a bulletin template | +1" 

and then the total score is 4....